Makefile

```diff
@@ -24,6 +24,7 @@ PULL ?=True
 DEV ?=y
 CHECK_CHANGED_SCRIPT=$(CWD)/changed_from_main.py
 CONTAINER_CMD?=docker
+SAVEPOINT_SETUP=pip3 list && python3 -m gt4py.gt_src_manager install
```

Comment:
I think at some point this was part of the dockerfile? Also, I see that this is called during set up environment in circleci, does that not work?

Reply:
The complicated case is that on the CircleCI test, we:
- Cannot run this outside of the container, because the environment is not set up / installed (I could add it just to run this command, but that feels like overkill)
- Are running the MPI-parallel tests, so if the process itself tries to clone these sources it happens 6 times in parallel, which leads to errors
- Need to bind-mount our directory into the container so that we can retain the caches, but doing so over-writes the gt4py directory (including gridtools sources) with our directory that does not have these

I could turn this into a `?=` and modify it only for the CircleCI tests, if that would be better? Normally this isn't an issue because we have/can get the gridtools sources on our local filesystem copy of gt4py.

Reply:
Okay, just to double check, if we already had gridtools installed, this wouldn't do anything right?